<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionSelectWidget</class>
 <widget class="QWidget" name="VersionSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="sideBox">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="filterBox" native="true">
       <layout class="QVBoxLayout" name="_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="releasesCheckbox">
          <property name="text">
           <string>Releases</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="snapshotsCheckbox">
          <property name="text">
           <string>Snapshots</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="oldSnapshotsCheckbox">
          <property name="text">
           <string>Old Snapshots</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="betasCheckbox">
          <property name="text">
           <string>Betas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="alphasCheckbox">
          <property name="text">
           <string>Alphas</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QProgressBar" name="loadProgressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="VersionListView" name="listView">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VersionListView</class>
   <extends>QTreeView</extends>
   <header>widgets/VersionListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
